build.sbt

```diff
@@ -20,6 +20,7 @@ lazy val paDigitaleReportGeneratorModuleName         = "padigitale-report-genera
 lazy val dashboardMetricsGeneratorModuleName         = "dashboard-metrics-report-generator"
 lazy val certifiedMailSenderModuleName               = "certified-mail-sender"
 lazy val certifiedMailSenderModelsModuleName         = "certified-mail-sender-models"
+lazy val eservicesFileGeneratorModuleName            = "eservices-file-generator"
```

Comment:
no strong opinions here, but having a specific purpose, we could name this job something like `eservices-monitoring-exporter`.
Any better names are welcome

Reply:
I agree with @galales